contracts/instruments/RibbonThetaVault.sol

```diff
@@ -159,15 +162,10 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
      * @notice Closes the vault and makes it withdraw only.
      */
     function sunset(address upgradeTo) external onlyOwner {
-        require(!isSunset, "!isSunset");
-        require(replacementVault == address(0), "!replacementVault");
+        require(address(replacementVault) == address(0), "Already sunset");
```

Comment:
`isSunset` is superfluous because checking if the `replacementVault == address(0)` serves the same purpose

Reply:
The additional check for was cases where vaults were closed w/o a V2 upgrade. Since that's no longer the case can remove